packages/extension-base/src/background/handlers/State.ts

```diff
@@ -59,17 +60,30 @@ interface SignRequest extends Resolver<ResponseSigning> {
 
 let idCounter = 0;
 
-const WINDOW_OPTS: chrome.windows.CreateData = {
-  // This is not allowed on FF, only on Chrome - disable completely
-  // focused: true,
+const NOTIFICATION_URL = chrome.extension.getURL('notification.html');
+
+const POPUP_WINDOW_OPTS: chrome.windows.CreateData = {
+  focused: true,
   height: 621,
   left: 150,
   top: 150,
```

Comment:
note that these settings are ultimately the culprit of a really bad behavior. I dont see a good reason to specify these window sizes, since users can just manually control it themselves.

Reply:
see: https://github.com/polkadot-js/extension/pull/768